apps/dashboard/src/pages/navbar-pages/staking/Validators.svelte

```diff
@@ -75,10 +74,6 @@
   $: totalReadyToClaim = $stakeInfo.then(getTotal('readyToClaim'))
 
   const dispatch = createEventDispatcher<{
-    'show-claim-all': undefined
-    'show-claim-single': string
-    'show-stake-multiple': undefined
-    'show-stake-single': string
```

Comment:
why remove these events? the point is to not have this component worry about such "goto" side effects which depend on the routes